src/Microsoft.PowerShell.CoreCLR.Eventing/DotNetCode/Eventing/EventProvider.cs

```diff
@@ -94,15 +94,20 @@ public EventProvider(Guid providerGuid)
         [System.Security.SecurityCritical]
         private unsafe void EtwRegister()
         {
-            uint status;
-
             _etwCallback = new UnsafeNativeMethods.EtwEnableCallback(EtwEnableCallBack);
 
+            #if !CORECLR   
```

Comment:
>!C [](start = 16, length = 2)

Let us use same strategy we are planning on using for PInvokeDllNames.cs file instead of removing it.